foedus-core/src/foedus/storage/array/array_storage_pimpl.cpp

```diff
@@ -51,7 +51,7 @@
 namespace foedus {
 namespace storage {
 namespace array {
-
+// FIXME(tzwang): overwrite_record here doesn't add to read set, should we?
```

Comment:
It's correct to not add to read-set. Overwrite is a blind write.
In other storage types, even overwrite is not blind because it has to check the existence of the record, but it's a genuine blind write in array storage.